po/fr.po

```diff
@@ -35,7 +35,7 @@ msgstr "N°"
 #: ../addons/src/ao_bookmarklist.c:350 ../devhelp/src/dhp-object.c:494
 #: ../devhelp/devhelp/dh-window.c:1142
 msgid "Contents"
-msgstr "Contenu"
+msgstr "Contenus"
```

Comment:
For the DevHelp usage, it is a left panel subtabs label. In this context, twice are possible. So, in the doubt, I rollback this string. Thanks